innofw/core/models/torch/lightning_modules/segmentation.py

```diff
@@ -133,6 +134,7 @@ def stage_step(self, stage, batch, do_logging=False, *args, **kwargs):
         #     predictions = torch.sigmoid(predictions)
 
         output[SegOutKeys.predictions] = predictions
+        self.training_metric_values.append(self.train_metrics.clone())
```

Comment:
self.training_metric_values is not needed and it occupies extra RAM. Better to remove it